app/docs/dev.md

```diff
@@ -1,29 +1,207 @@
+| are              | $10^2~\mathrm{metre}^2$                              |
+| hectare          | $10^4~\mathrm{metre}^2$                              |
+| bar              | $10^5~\mathrm{pascal}$                               |
+| barn             | $10^{-28}~\mathrm{metre}$                            |
+| curie            | $3.7 \cdot 10^{10}~\mathrm{becquerel}                |
+| roentgen         | $2.58 \cdot 10^{-4}~\mathrm{kelvin~(kilogram)}^{-1}$ |
+| rad              | $10^{-2}~\mathrm{gray}$                              |
+| rem              | $10^{-2}~\mathrm{sievert}$                           |
+
+#### Table: Imperial units
+
+Commonly used imperial units taken from [Wikipedia: Imperial units](https://en.wikipedia.org/wiki/Imperial_units)
+
+| Unit name         | Symbol | Expressed in SI units                         |
+|-------------------|:-------|:----------------------------------------------|
+| inch              |   in   | $0.0254~\mathrm{metre}$                       |
+| foot              |   ft   | $0.3048~\mathrm{metre}$                       |
+| yard              |   yd   | $0.9144~\mathrm{metre}$                       |
+| mile              |   mi   | $1609.344~\mathrm{metre}$                     |
+| fluid ounce       |  fl oz | $28.4130625~\mathrm{millilitre}$              |
+| gill              |   gi   | $142.0653125~\mathrm{millilitre}$             |
+| pint              |   pt   | $568.26125~\mathrm{millilitre}$               |
+| quart             |   qt   | $1.1365225~\mathrm{litre}$                    |
+| gallon            |   gal  | $4546.09~\mathrm{litre}$                      |
+| ounce             |   oz   | $28.349523125~\mathrm{gram}$                  |
+| pound             |   lb   | $0.45359237~\mathrm{kilogram}$                |
+| stone             |   st   | $6.35029318~\mathrm{kilogram}$                |
+
+### `strict_syntax`
+
+If `strict_syntax` is set to true then the answer and response must have `*` or `/` between each part of the expressions and exponentiation must be done using `**`, e.g. `a*b/c**2` is accepted but `ab/c^2` is not.
+
+If `strict_syntax` is set to false, then `*` can be omitted and `^` used instead of `**`, e.g. `a*b/c**2` and `ab/c^2` will be considered identical. In this case it is also recommended to list any multicharacter symbols expected to appear in the response as input symbols.
+
+By default `strict_syntax` is set to true.
 
 ## Examples
-*List of example inputs and outputs for this function, each under a different sub-heading*
-
-### Simple Evaluation
-
-```python
-{
-  "example": {
-    "Something": "something"
-  }
-}
-```
-
-```python
-{
-  "example": {
-    "Something": "something"
-  }
-}
-```
\ No newline at end of file
+
+Implemented versions of these examples can be found in the module 'Examples: Evaluation Functions'.
+
+### 1 Checking if a set of quantities match the Buckingham pi theorem
```

Comment:
Is there only '1'? In which case remove the number 1, and relabel a/b/c/d as 1/2/3/4?

Reply:
Original labeling was chosen to match task in examples module, changed labelling in both examples module and documentation as suggested.